prototypes/ahmed_body_simplex/ahmed.prm

```diff
@@ -0,0 +1,115 @@
+# Listing of Parameters
+# ---------------------
+
+
+# --------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set output frequency        = 1
+  set output name             = ahmed-output
+  set output path             = ./
+  set time end                = 4
+  set time step		      = 0.01
+end
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+end
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+subsection timer
+    set type    = iteration                                  # <none|iteration|end>
+end
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection uvwp
+            set Function expression = 1; 0; 0
+    end
+end
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+    set kinematic viscosity         = 1e-3
+end
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = gmsh
+    set file name            = Ahmed_Body_20_2D.msh
+    set enable simplex       = true
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 3
+    subsection bc 0
+        set type              = noslip
+    end
+    subsection bc 1
+        set type              = function
+        subsection u
+            set Function expression = 1
+        end
+        subsection v
+            set Function expression = 0
+        end
+        subsection w
+            set Function expression = 0
+        end
+    end
+    subsection bc 2
+        set type              = slip
+    end
+end
+
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = kelly
+  set variable                = velocity
+  set fraction type           = number
+  set max number elements     = 70000
+  set max refinement level    = 5
+  set min refinement level    = 0
+  set frequency               = 5
+  set fraction refinement     = 0.3
+  set fraction coarsening     = 0.1
+end
```

Comment:
Put mesh adaptation to none. No mesh adaptation should be used when simplex mesh is on.